src/applications/personalization/view-representative/components/ViewRepresentativeDetails.jsx

```diff
@@ -1,11 +1,39 @@
 import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
 
 export const ViewRepresentativeDetails = props => (
   <>
-    <h2 className="vads-u-margin-top--1p5">
-      Your selected representative for VA claims
-    </h2>
-    <div className="vads-u-background-color--gray-lightest vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-padding-top--1 vads-u-padding-bottom--2 vads-u-padding-x--2 vads-u-font-size--md vads-u-font-family--sans">
+    <h2 className="vads-u-font-size--h3 vads-u-margin-top--6">On this page</h2>
+    <ul className="usa-unstyled-list vads-u-margin-top--2">
+      <li>
+        <a href="#selectedRepresentative">
+          <i className="fas fa-arrow-down va-c-font-size--xs vads-u-margin-top--1 vads-u-margin-right--1" />
+          Your selected representative for claims
+        </a>
+      </li>
+      <li className="vads-u-margin-top--3">
+        <a href="#whatCanRepresentativeDo">
+          <i className="fas fa-arrow-down va-c-font-size--xs vads-u-margin-top--1 vads-u-margin-right--1" />
+          What can a representative do?
+        </a>
+      </li>
+      <li className="vads-u-margin-top--3">
+        <a href="#whatDoesItCost">
+          <i className="fas fa-arrow-down va-c-font-size--xs vads-u-margin-top--1 vads-u-margin-right--1" />
+          What does it cost to use a recognized representative or a VSO?
+        </a>
+      </li>
+      <li className="vads-u-margin-top--3">
+        <a href="">
+          <i className="fas fa-arrow-down va-c-font-size--xs vads-u-margin-top--1 vads-u-margin-right--1" />
+          Need help?
+        </a>
+      </li>
+      <h2 id="selectedRepresentative" className="vads-u-margin-top--5">
```

Comment:
This needs to be outside of the unordered list.

Reply:
Updated this.